Makefile

```diff
@@ -151,21 +151,24 @@ helm-chart-generate: kustomize helm kubectl-slice yq charts
 	for file in charts/$(CHART_NAME)/raw-files/*rolebinding*; do\
 		$(YQ) -i '.subjects[0].namespace = "{{ .Release.Namespace }}"' $${file};\
 	done
-	# Add .spec.replicas for the controller-manager deployment
-	for file in charts/$(CHART_NAME)/raw-files/deployment-*-controller-manager.yaml; do\
-		$(YQ) -i '.spec.replicas = "{{ (.Values.Operator).replicas | default 1 }}"' $${file};\
-	done
+
 	# Correct .metadata.name for cluster scoped resources
 	cluster_scoped_files="charts/$(CHART_NAME)/raw-files/clusterrolebinding-awx-operator-proxy-rolebinding.yaml charts/$(CHART_NAME)/raw-files/clusterrole-awx-operator-metrics-reader.yaml charts/$(CHART_NAME)/raw-files/clusterrole-awx-operator-proxy-role.yaml";\
 	for file in $${cluster_scoped_files}; do\
 		$(YQ) -i '.metadata.name += "-{{ .Release.Name }}"' $${file};\
 	done
 	# Correct the reference for the clusterrolebinding
 	$(YQ) -i '.roleRef.name += "-{{ .Release.Name }}"' 'charts/$(CHART_NAME)/raw-files/clusterrolebinding-awx-operator-proxy-rolebinding.yaml'
-	# Correct .spec.replicas type for the controller-manager deployment
+
+	# Feed controller deployment file into template to allow for override from values
```

Comment:
@miles-w-3 I think this is a good change, but it may be better to call this parameter something other than `controller` because it may cause confusion with the downstream of awx (which is called Automation Controller and referred to as controller).  

I know this is referring to the k8s controller-manager (how k8s docs refer to the operator deployment)... But in our unique position, maybe we could call it `operator-controller`, or just `operator`? , or even `manager`, as that is what the deployment yaml file is called (config/manager/manager.yaml).  

What do you think?

Reply:
I defer to your judgement, operator-controller or manager-deployment make sense to me and I can make the change. Can also update the template names in the chart to avoid confusion

Reply:
@rooftopcellist I have updated the pr changing the key to `operator-controller`

Reply:
@miles-w-3 Thanks for making the change. That looks good.